tests/packagedcode/data/pypi/requirements.txt/gemfileparser-requirements-txt-expected.json

```diff
@@ -0,0 +1,52 @@
+{
+  "type": "pypi",
+  "namespace": null,
+  "name": null,
+  "version": null,
+  "qualifiers": {},
+  "subpath": null,
+  "primary_language": "Python",
+  "description": null,
+  "release_date": null,
+  "parties": [],
+  "keywords": [],
+  "homepage_url": null,
+  "download_url": null,
+  "size": null,
+  "sha1": null,
+  "md5": null,
+  "sha256": null,
+  "sha512": null,
+  "bug_tracking_url": null,
+  "code_view_url": null,
+  "vcs_url": null,
+  "copyright": null,
+  "license_expression": null,
+  "declared_license": null,
+  "notice_text": null,
+  "root_path": null,
+  "dependencies": [
+    {
+      "purl": "pkg:pypi/setuptools@32.0.0",
+      "requirement": "==32.0.0",
```

Comment:
This is not correct. I would expect this value to be `setuptools`

Reply:
@majurg Please see https://github.com/nexB/scancode-toolkit/pull/2077#discussion_r443118795 comment of @pombredanne

Reply:
Ok. If `requirement` is not used for anything (now that all the package information is in the purl), why not delete this field? 

I guess mainly I am confused at to what `requirement` refers to. I interpret that field name to mean the name of the python package, but maybe this is wrong?

Reply:
```
purl = String(
    repr=True,
    label='Dependent package URL',
    help='A compact purl package URL. Typically when there is an unresolved requirement, there is no version. ' 'If the dependency is resolved, the version should be added to the purl')

requirement = String(
    repr=True,
    label='dependent package version requirement',
    help='A string defining version(s)requirements. Package-type specific.')
```
So both are important :)

Reply:
Ah ok this is fine then :+1: thanks